systems/plants/dev/RigidBodySpatialForce.m

```diff
@@ -0,0 +1,53 @@
+classdef RigidBodySpatialForce < RigidBodyForceElement
+  
```

Comment:
notice that this is in dev.  didn't write the test case yet.  and would benefit from a conversation with @tkoolen about whether i'm using his best tools (and even getting the transforms correct :)